zappa/core.py

```diff
@@ -1126,14 +1129,27 @@ def update_lambda_function(self, bucket, function_name, s3_key=None, publish=Tru
         resource_arn = response['FunctionArn']
         version = response['Version']
 
-        # Given an alias name, let's update that alias to map to this newly
-        # updated version.
+        # If the lambda has an ALB alias, let's update the alias
+        # to point to the newest version of the function.
         # Related: https://github.com/Miserlou/Zappa/pull/1730
-        self.lambda_client.update_alias(
-            FunctionName=function_name,
-            FunctionVersion=version,
-            Name=ALB_LAMBDA_ALIAS,
-        )
+        #          https://github.com/Miserlou/Zappa/issues/1823
+        try:
```

Comment:
When I first saw how this was determining if an alias already existed I went to the AWS API docs to see if there was an API for testing if an alias exists e.g. HEAD /some/endpoint/alias as opposed to GET /home/endpoint/alias with the intent of avoiding the need for the try-catch block. 

It seems there is no such HEAD endpoint, and maybe everybody else is familiar with that, but it may be worth leaving a comment here explaining that.

Reply:
Seems reasonable. Let me add that in.